### ISSUE TITLE
Tables support

### ISSUE THREAD

**Issue by fabioebner** [iliyaZelenko/tiptap-vuetify#20]
Do you have plans to support tables?

**Comment by iliyaZelenko** [comment]
Yes, definitely. I plan to support all the capabilities that are in the tipup.

I also want to note that I am developing a system with which you can make your extensions for this package.

Unfortunately, at the moment I have nuances with my main job, but as soon as everything gets better, I will begin to actively pursue development. One big version is almost ready.

**Comment by dsheehan101** [comment]
Hi @iliyaZelenko ,
Would you still have plans to add integration of Tables into this project?
If not, would you perhaps have a good example of how to add this?

Many thanks

**Comment by iliyaZelenko** [comment]
@dsheehan101 by likes, I see many people need this feature. Then I will try to do it for you soon.

In theory, you can add any type extension through [native-extensions](https://github.com/iliyaZelenko/tiptap-vuetify#native-extensions). Already somewhere there was more detailed information on how to do this.

**Comment by iliyaZelenko** [comment]
Basic support has been added. [Here is an example](https://github.com/iliyaZelenko/tiptap-vuetify-demo/commit/3147c06e79ca64041f90bab0e0338ca950ee6804) of how to add tables to the editor.

But it remains to be done:
1) So that the text can be translated into different languages
2) Different actions with tables. For example, merge cells.

**Comment by apostolov1997** [comment]
Hi @iliyaZelenko,  
Seems like the table support is almost ready but unfinished. The button icon shows but isn't aligned correctly and the tooltip text says "TODO".
Please see ![attached image](https://user-images.githubusercontent.com/35740450/116571696-984f4b80-a90b-11eb-923f-90672935f735.png)
